# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLTypes.h
# Commit: 0b191b707750
# Full Hash: 0b191b707750e040289033359306e4966c1bf062
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - WebXR compositing fixes. r=imanol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78117
# ==============================================================================

diff -r 16c5dd2cfd60 -r 0b191b707750 dom/canvas/WebGLTypes.h
--- a/dom/canvas/WebGLTypes.h	Mon Jun 15 18:25:55 2020 +0000
+++ b/dom/canvas/WebGLTypes.h	Mon Jun 15 18:25:58 2020 +0000
@@ -990,26 +990,6 @@
 }
 
 // -
-// ClientWebGLContext exports!
-// (So that you don't need to include the mess that is ClientWebGLContext.h)
-namespace gfx {
-class SourceSurface;
-}  // namespace gfx
-
-namespace layers {
-class KnowsCompositor;
-class SurfaceDescriptor;
-}  // namespace layers
-
-namespace webgl {
-
-void Present(ClientWebGLContext&);
-Maybe<layers::SurfaceDescriptor> GetFrontBuffer(ClientWebGLContext&,
-                                                layers::KnowsCompositor*);
-
-}  // namespace webgl
-
-// -
 
 }  // namespace mozilla
 